test/go-tests/testsuite_gke_test.go

```diff
@@ -17,6 +17,7 @@ func Test_GKE(t *testing.T) {
 	t.Run("Test_LogForwarding", Test_LogForwarding)
 	t.Run("Test_SequenceState", Test_SequenceState)
 	t.Run("Test_SequenceStateParallelStages", Test_SequenceStateParallelStages)
+	t.Run("Test_SequenceStateParallelServices", Test_SequenceStateParallelServices)
```

Comment:
why only GKE? I think we should run them for all platforms

Reply:
true I'll copy it everywhere :)